src/Application/Features/InternshipVacancies/Command/Create/CreateInternshipVacancyCommandHandler.cs

```diff
@@ -36,5 +37,7 @@ public async Task<Guid> Handle(CreateInternshipVacancyCommand request, Cancellat
         await _context.VacancyTimelineEvents.AddAsync(timelineEvent, cancellationToken);
         await _context.SaveChangesAsync(cancellationToken);
         return internship.Id;
-    }
+    }
+
+    private string GenerateSlug(string title) => $"internships-{title.ToLower().Trim()}";
```

Comment:
Using a title is enough. with internship prefix, it will not looks good. For example: 
.../internship/internship-summer2023

It would looks better when we just do: 
.../internship/summer2023